components/FormCard.tsx

```diff
@@ -169,12 +169,7 @@ export const FormCard: React.FC<FormCardProps> = ({
         {submitError && <ErrorMessage error={submitError} />}
         <div className={`text-center align-self-${textAlignment}`}>
           {newBtn ? (
-            <button
-              onClick={btnOnClick}
-              className={styles['formCard--submitButton']}
-            >
-              {onSubmit.title}
-            </button>
+            <NewButton onClick={btnOnClick}>{onSubmit.title}</NewButton>
```

Comment:
I wasn't sure what to call the new button, so I just named it NewButton since the `newBtn` variable was used to refer to it.
Other possible names we could have used: MaterialButton, ResponsiveButton
Let me know if you think of a good name.
We can always change the name in the future if we think of a better name.

Reply:
`NewButton` is good.